src/containers/shared/components/Transaction/NFTokenAcceptOffer/parser.ts

```diff
@@ -18,6 +20,29 @@ export const parser: TransactionParser<
     acceptedOfferIDs.push(tx.NFTokenSellOffer)
   }
 
+  // If in brokered mode, we must fetch both of the NFTokenOffer nodes
+  // in order to fetch the seller and buyer from each
+  if (acceptedOfferNodes.length > 1) {
+    const buyOfferNode = acceptedOfferNodes.find(
+      (node: any) =>
+        !determineIsSellOffer(node.DeletedNode?.FinalFields?.Flags),
+    )?.DeletedNode?.FinalFields
+
+    const sellOfferNode = acceptedOfferNodes.find((node: any) =>
+      determineIsSellOffer(node.DeletedNode?.FinalFields?.Flags),
+    )?.DeletedNode?.FinalFields
+
+    return {
+      amount: formatAmount(buyOfferNode.Amount),
+      tokenID: buyOfferNode.NFTokenID,
+      seller: sellOfferNode.Owner,
+      buyer: buyOfferNode.Owner,
+      acceptedOfferIDs,
+    }
+  }
+
+  const acceptedOfferNode = acceptedOfferNodes[0]?.DeletedNode?.FinalFields
```

Comment:
What will be showin if the accept fails and there is no offer in the meta data?  Will it show the wrong value for the buyer?

Reply:
The logic would be identical as before. The conditional check right after will be responsible for handling undefined if node doesnt exist.
`  if (!acceptedOfferNode) {
    return {
      acceptedOfferIDs,
    }
  }`